public func foo() -> Bool { fatalError() }
